### PR TITLE
Adding ICS CAN FD support

### PR DESCRIPTION
**Pull request by pierreluctg**

### CODE REVIEW

File: can/interfaces/ics_neovi/neovi_bus.py [hunk 1]
```diff
@@ -84,6 +84,8 @@ def __init__(self, channel, can_filters=None, **config):
         :param int bitrate:
             Channel bitrate in bit/s. (optional, will enable the auto bitrate
             feature if not supplied)
+        :param int fd_bitrate:
```
[the remark this answers is not in the document]
Comment (christiansandberg) [reply can/interfaces/ics_neovi/neovi_bus.py]: In the other interfaces we have used `data_bitrate` to refer to the bitrate used during the data phase.

File: can/interfaces/ics_neovi/neovi_bus.py [hunk 2]
```diff
@@ -225,6 +230,7 @@ def _ics_msg_to_message(self, ics_msg):
             extended_id=bool(
                 ics_msg.StatusBitField & ics.SPY_STATUS_XTD_FRAME
             ),
+            is_fd=ics_msg.Protocol == ics.SPY_PROTOCOL_CANFD,
```
Comment (christiansandberg) [inline can/interfaces/ics_neovi/neovi_bus.py]: Is it possible to get the bitrate_switch and error_state_indicator too?

**Comment by christiansandberg** [comment]
Can you send CAN-FD frames too?

**Comment by pierreluctg** [comment]
@christiansandberg yes sending is also supported.

**Comment by felixdivo** [comment]
I don't get why Codecov complains here ... [This line](https://github.com/hardbyte/python-can/blob/develop/.codecov.yml#L14) should allow the coverage to drop that much before failing.

**Pull request merged**